src/util/schema_keywords.js

```diff
@@ -0,0 +1,165 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+const js_utils = require('./js_utils');
+const { _: codegen} = require('ajv');
+
+/**
+ * @typedef {import('ajv').KeywordCxt} KeywordCxt
+ */
+
+const KEYWORDS = js_utils.deep_freeze({
+
+    // schema: { date: true } will match (new Date())
+    date: {
+        keyword: 'date',
+        errors: false,
+        statements: true,
```

Comment:
Are these relevant still?? I don't think so.

Reply:
Ah ok, so errors: false is still relevant as written in the docs:
https://ajv.js.org/keywords.html#reporting-errors

But statements is irrelevant.

Reply:
Removed the statements